.changeset/twenty-rice-prove.md

```diff
@@ -0,0 +1,8 @@
+---
+"ledger-live-desktop": minor
+"live-mobile": minor
```

Comment:
Not sure but I think you need to split the changeset in two because you update both llm and lld

Reply:
https://github.com/LedgerHQ/ledger-live/wiki/Changesets